examples/node-device-info.js

```diff
@@ -0,0 +1,10 @@
+const ledger = require('../src/index');
+ledger
+	.comm_node
+	.create_async()
+	.then(function(comm) {
+		console.log(comm.device.getDeviceInfo());
+	})
+	.catch(function(reason) {
```

Comment:
This project currently uses `Q` promises, so there is no `catch`:

```javascript
.fail(function(reason) {
...
}
```

Reply:
Q does actually support the standard `.catch`: https://github.com/kriskowal/q/wiki/API-Reference#promisecatchonrejected. I prefer keeping it this way because most Promise implementations support it as a standard.

Reply:
I wasn't aware of that - good to know :)

Reply:
no worries! thanks @antony